src/main/java/training/chessington/model/pieces/Knight.java

```diff
@@ -15,6 +16,40 @@ public Knight(PlayerColour colour) {
 
     @Override
     public List<Move> getAllowedMoves(Coordinates from, Board board) {
-        return new ArrayList<>();
+        List<Move> allowableMoves = new ArrayList<>();
+
+        // laying rectangle
+        getCornersOfRectangle(from, board, allowableMoves, 1, 2);
+        // standing rectangle
+        getCornersOfRectangle(from, board, allowableMoves, 2, 1);

```

Comment:
In general, you want to avoid using comments in your code unless absolutely necessary. Normally your code should be written in a clean enough way that any coder should be able to look at it and understand it, without needing any comments